HW6_Starter_Code/schema.sql

```diff
@@ -1,2 +1,37 @@
 -- Insert code to create Database Schema
 -- This will create your .db database file for use
+DROP TABLE customer;
+DROP TABLE address;
+DROP TABLE orders;
+DROP TABLE customer_orders;
+
```

Comment:
The tables are missing all foreign key references to create relations between different tables